.github/workflows/service_test_gcs.yml

```diff
@@ -0,0 +1,31 @@
+name: Service Test Gcs
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - "docs/**"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  gcs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Test
+        shell: bash
+        run: cargo test gcs --features compress,retry -- --nocapture
+        env:
+          RUST_BACKTRACE: full
+          RUST_LOG: debug
+          OPENDAL_GCS_TEST: ${{ secrets.OPENDAL_GCS_TEST }}
+          OPENDAL_GCS_ROOT: ${{ secrets.OPENDAL_GCS_ROOT }}
```

Comment:
Should add OPENDAL_GCS_ENDPOINT

Reply:
Negative. We intend to use the default endpoint. It's the most common case for end user.